lib/components/Button/base/index.tsx

```diff
@@ -83,7 +85,7 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
                         className="quill-button__loader-icon"
                     />
                 )}
-                {children && <div>{children}</div>}
```

Comment:
@meenakshi-deriv is there any reason we have a div to wrap the children? Because of this, the icon are not aligned correctly. After I remove the div Button component is still working correctly.